.github/workflows/deploy.yml

```diff
@@ -0,0 +1,74 @@
+name: build-scratch-blocks
+on: 
+    push: 
+        branches: [master, develop, beta, hotfix/*]
+jobs:
+    setup:
+      runs-on: ubuntu-latest
+      env:
+        JVM_OPTS: -Xmx3200m
+        DETECT_CHROMEDRIVER_VERSION: "true"
+        PROJECT_PATH: ./scratch-blocks
+      steps:
+        - uses: actions/checkout@v3
+        - name: Install Chrome Dependencies
+          run: |
+            sudo apt-get update
+            sudo apt-get install -y libgconf-2-4 libatk1.0-0 libatk-bridge2.0-0 libgdk-pixbuf2.0-0 libgtk-3-0 libgbm-dev libnss3-dev libxss-dev libasound2
+        - name: Check Python version
+          run: python --version 
+        - name: Setup Java
+          uses: actions/setup-java@v3
+          with:
+            distribution: 'temurin'
+            java-version: 17
+        - name: Setup Node
+          uses: actions/setup-node@v3
+          with:
+            node-version-file: '.nvmrc'
+        - uses: browser-actions/setup-chrome@v1
+          with:
+            chrome-version: stable
+          id: setup-chrome
+        - run: |
+            echo Installed chromium version: ${{ steps.setup-chrome.outputs.chrome-version }}
+            ${{ steps.setup-chrome.outputs.chrome-path }} --version
+        - name: Install Chrome Driver
+          run: DETECT_CHROMEDRIVER_VERSION=true npm install chromedriver
```

Comment:
The `npm ci` step will actually delete anything installed by `npm install`, but since you have `DETECT_CHROMEDRIVER_VERSION` set in the job-global `env` section above it'll affect the `npm ci` here, so you can just remove the `npm install` step.
```suggestion
```